setup/pattoo_installation.py

```diff
@@ -214,11 +231,78 @@ def systemd(self, width=80):
             None
         """
         # Initialize key variables
-        _ = self.subparsers.add_parser(
+        parser = self.subparsers.add_parser(
             'systemd',
             help=textwrap.fill('Install systemd service files', width=width)
         )
 
+        # Add arguments
+        parser.add_argument(
+            '--verbose',
+            action='store_true',
+            help='Enable verbose mode.')
+
+
+def get_pattoo_home():
+    """Retrieve home directory for pattoo user.
+
+    Args:
+        None
+
+    Returns:
+        The home directory for the pattoo user
+
+    """
+    try:
+        # No exception will be thrown if the pattoo user exists
```

Comment:
Only have one action in a try clause. The one you feel will most likely fail. Put the rest outside of it. You have no way of knowing which of these statements will cause a potential error, hence it's not very useful.